src/NzbDrone.Core.Test/MetadataSource/SkyHook/SkyHookProxySearchFixture.cs

```diff
@@ -70,13 +70,30 @@ public void successful_search(string title, string expected)
             ExceptionVerification.IgnoreWarns();
         }
 
+
+        [TestCase("Evolve", "Imagine Dragons", "Evolve")]
+        [TestCase("Hysteria", null, "Hysteria")]
+        [TestCase("lidarr:d77df681-b779-3d6d-b66a-3bfd15985e3e", null, "Pyromania")]
+        [TestCase("lidarr: d77df681-b779-3d6d-b66a-3bfd15985e3e", null, "Pyromania")]
+        [TestCase("lidarrid:d77df681-b779-3d6d-b66a-3bfd15985e3e", null, "Pyromania")]
+        public void successful_album_search(string title, string artist, string expected)
```

Comment:
See other comment about 400 on invalid id. Artists can also return 403 if blacklisted. I now see some cases below with invalid. Maybe similar should be added for albums?